### PR TITLE
Added velocity calculation

### PR DESCRIPTION
**Pull request by lironbc**

### CODE REVIEW

File: root/backend/app.js [hunk 1]
```diff
@@ -76,9 +92,61 @@ app.get('/oauth/register', (req, res) => {
...
+
+// function setRaptorName(raptor_name) {
+//     database.query(
+//         `UPDATE users SET raptor_name = '${raptor_name}' WHERE id ='${username}'`,
+//         (err, rows, fields) => {
+//             if (err) throw err;
+//             //console.log('table updated...');
+//         }
+//     );
+// }
```
Comment (AndrewKassab) [inline root/backend/app.js:144]: Reasons for commenting everything out? Probably better to just remove all the code if its not necessary anymore. We can always check out to a past commit if it's a safety net

File: root/backend/database.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// app.get('/updatetable', (req, res) => {
+//     let sql = "UPDATE table2 SET text = 'this is 3rd' WHERE id = 3";
+//     db.query(sql, (err, rows, fields) => {
+//         if (err) throw err;
+//         console.log('table updated...');
+//         res.send('table updated...');
+//     });
+// });
+
+// module.exports = db;
```
Comment (AndrewKassab) [inline root/backend/database.js:74]: Same question with this file or if this is something we are supposed to be using in the future

File: root/src/js/github/issues.js [hunk 3]
```diff
@@ -61,6 +62,8 @@ function getGithubIssues(username) {
                         listElement.appendChild(link);
 
                         issueList.appendChild(listElement);
+
+                        // speed.innerHTML = "Raptor's Speed: 25km/h";
```
Comment (AndrewKassab) [inline root/src/js/github/issues.js:66]: same remove since we are calculating it now right

**Comment by KenLHua** [comment]
move to dev master

**Pull request closed without merging**